build/source/driver/multi_driver.f90

```diff
@@ -128,70 +143,78 @@ program multi_driver
-integer(i4b)              :: jStep=0                        ! index of model output
+type(gru_hru_double)             :: dparStruct                 ! x%gru(:)%hru(:)%var(:)     -- default model parameters
+! define indices
+integer(i4b)                     :: iVar                       ! index of a model variable 
+integer(i4b)                     :: iStruct                    ! loop through data structures
+integer(i4b)                     :: iGRU
+integer(i4b)                     :: iHRU,jHRU,kHRU             ! index of the hydrologic response unit
+integer(i4b)                     :: nGRU                       ! number of grouped response units
+integer(i4b)                     :: nHRU                       ! number of global hydrologic response units
+integer(i4b)                     :: hruCount                   ! number of local hydrologic response units
+integer(i4b)                     :: iStep=0                    ! index of model time step
+integer(i4b)                     :: jStep=0                    ! index of model output
+integer(i4b)                     :: ix_gru                     ! index of GRU that corresponds to the global HRU
+integer(i4b)                     :: ix_hru                     ! index of local HRU that corresponds to the global HRU
+integer(i4b)                     :: jx_gru                     ! mapping from a previously used forcing file: index of GRU that corresponds to the global HRU
+integer(i4b)                     :: jx_hru                     ! mapping from a previously used forcing file: index of local HRU that corresponds to the global HRU
 ! define the re-start file
-logical(lgt)              :: printRestart                   ! flag to print a re-start file
-integer(i4b),parameter    :: ixRestart_iy=1000              ! named variable to print a re-start file once per year
-integer(i4b),parameter    :: ixRestart_im=1001              ! named variable to print a re-start file once per month
-integer(i4b),parameter    :: ixRestart_id=1002              ! named variable to print a re-start file once per day
-integer(i4b),parameter    :: ixRestart_never=1003           ! named variable to print a re-start file never
-integer(i4b)              :: ixRestart=ixRestart_im         ! define frequency to write restart files
+logical(lgt)                     :: printRestart               ! flag to print a re-start file
+integer(i4b),parameter           :: ixRestart_iy=1000          ! named variable to print a re-start file once per year
+integer(i4b),parameter           :: ixRestart_im=1001          ! named variable to print a re-start file once per month
+integer(i4b),parameter           :: ixRestart_id=1002          ! named variable to print a re-start file once per day
+integer(i4b),parameter           :: ixRestart_never=1003       ! named variable to print a re-start file never
+integer(i4b)                     :: ixRestart=ixRestart_im     ! define frequency to write restart files
```

Comment:
Isn't this set from the config files? If not, then we should open an issue to change that (no need to change as part of this PR).